internal/cli/iam/organizations/invitations/invite.go

```diff
@@ -70,12 +71,14 @@ func InviteBuilder() *cobra.Command {
 	opts.Template = createTemplate
 	cmd := &cobra.Command{
 		Use:     "invite <email>",
-		Short:   "Invites one user to the organization that you specify.",
+		Short:   "Invite the specified MongoDB user to your organization.",
```

Comment:
just in case this is cloud user and not db user, mongodb user let's this a bit ambiguous